cpp/include/cudf/column/column_factories.hpp

```diff
@@ -239,10 +239,8 @@ std::unique_ptr<column> make_fixed_width_column(
  * @param strings The vector of pointer/size pairs.
  *                Each pointer must be a device memory address or `nullptr`
  * (indicating a null string). The size must be the number of bytes.
- * @param stream Optional stream for use with all memory allocation
- *               and device kernels
- * @param mr Optional resource to use for device memory
- *           allocation of the column's `null_mask` and children.
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ * @param mr Device memory resource used for allocation of the column's `null_mask` and children.
```

Comment:
Doesn't it also allocates memory for column's `data` ?

Reply:
string column's data is in children. (offsets_column and chars_column).

Reply:
My apologies, I saw function declaration on top for `fixed_width_column` and got confused.